frontend/src/validations/validateCategory.ts

```diff
@@ -1,8 +1,14 @@
 import { InputStatus } from '@/components/common/Input/Input.type';
 
-export const validateCategory = (category: string, isCategoryExist: (category: string) => boolean) => {
-  if (category.length > 8) return { status: 'ERROR' as InputStatus, message: '8자 이하로 입력해주세요' };
-  if (isCategoryExist(category)) return { status: 'ERROR' as InputStatus, message: '중복된 카테고리 입니다.' };
+import { DEFAULT_CATEGORY_VALUE } from '@/hooks/PairRoom/useCategories';
+
+const MAX_CATEGORY_LENGTH = 10;
```

Comment:
최대 카테고리 갯수인지, 이름 길이인지 조금 불명확할 수 있을 것 같아요!
```suggestion
const MAX_CATEGORY_NAME_LENGTH = 10;
```